tasks.py

```diff
@@ -157,17 +157,17 @@ def _detect_space(repo, branch=None, yes=False):
 
 
 SPACE_URLS = {
-    'dev': [('18f.gov', 'fec-dev-api')],
-    'stage': [('18f.gov', 'fec-stage-api')],
-    'prod': [('18f.gov', 'fec-prod-api')],
+    'dev': [('app.cloud.gov', 'fec-dev-api')],
+    'stage': [('app.cloud.gov', 'fec-stage-api')],
+    'prod': [('app.cloud.gov', 'fec-prod-api')],
```

Comment:
I don't think so since we are still using the proxy app. We need a proxy anyway so I think keeping the proxy as is makes sense. We could change it later if there is some benefit down the road.